docs/guide/install-and-run.asciidoc

```diff
@@ -113,6 +113,12 @@ NOTE: Check the <<agent-server-compatibility,Agent/Server compatibility matrix>>
 |{apm-java-ref}/supported-technologies-details.html[Supported technologies]
 |{apm-java-ref}/configuration.html[Config]
 
+.2+|.NET
+3+|The .NET agent comes with ASP.NET Core support out of the box.
```

Comment:
I usually say we have "auto instrumentation for ASP.NET Core". Because the agent API also works with Full Framework (the old stuff), it's just we don't have atm any auto instrumentation capability for that... So it'd be nice to somehow hint that the agent works on every .NET Flavor, but autoinstrumentation is currently ASP.NET Core only.

Reply:
Gotcha. What about something like:

> The .NET agent automatically instruments ASP.NET Core, while the Full framework can be instrumented with the public API.

Reply:
Pushed a commit with that change ⬆️

I'm going to merge this once CI passes so I can get the backports merged in time for the release. If there are still changes you'd like to see, just let me know! I have no problem opening up a new PR with your suggestions.

Reply:
> Gotcha. What about something like:
> 
> > The .NET agent automatically instruments ASP.NET Core, while the Full framework can be instrumented with the public API.

This sounds good to me 👍

Reply:
The official terminology is just `.NET Framework` and `.NET Core`. Here is [an example in Microsoft docs](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.trace.write?view=netframework-4.7.2#applies-to). So I wonder if we should use `Full framework` term. Instead of

>The .NET agent automatically instruments ASP.NET Core, while the Full framework can be instrumented with the public API.

I would say something like

> The .NET agent automatically instruments ASP.NET Core applications, while .NET Framework ASP.NET applications can be instrumented with the public API.

Reply:
Oooo good catch @SergeyKleyman. We should probably use the official terminology.

Reply:
+1 for using `.NET Framework`, but would not say "while .NET Framework ASP.NET applications can", because that not only applies to ASP.NET... it can be anything. 

So I'd propose:
>  The .NET agent automatically instruments ASP.NET Core applications, while .NET Framework  applications can be instrumented with the public API.

(or "applications running on top of .NET Framework")

Reply:
This sounds good to me 👍

Reply:
Boom! Look at that. Who says GitHub can't be synchronous 😂. Thanks to both of you 😃